api/capi/src/nnstreamer-capi-single-new.c

```diff
@@ -450,6 +451,7 @@ ml_single_open (ml_single_h * single, const char *model,
    * 3. Construct a pipeline
    * Set the pipeline desc with nnfw.
    */
+  single_h->nnfw = nnfw;
```

Comment:
Is there any possibility to overwrite `single_h->nnfw` when this open API is called twice?
Then, it's better to check this at the beginning.

Reply:
With every open to `ml_single_open`, a new `single_h` object is made. So, if this open API is called twice, two `single_h` objects will be allocated and returned to the user.
So, I think overwrite should not happen.

Reply:
Oh, okay